quantum/split_common/serial.h

```diff
@@ -1,21 +1,18 @@
 #ifndef MY_SERIAL_H
 #define MY_SERIAL_H
 
-#include "config.h"
 #include <stdbool.h>
 
-/* TODO:  some defines for interrupt setup */
-#define SERIAL_PIN_DDR DDRD
-#define SERIAL_PIN_PORT PORTD
-#define SERIAL_PIN_INPUT PIND
-#define SERIAL_PIN_MASK _BV(PD0)
-#define SERIAL_PIN_INTERRUPT INT0_vect
-
-#define SERIAL_SLAVE_BUFFER_LENGTH MATRIX_ROWS/2
-#define SERIAL_MASTER_BUFFER_LENGTH 1
-
-// Address location defines 
-#define SERIAL_BACKLIT_START   0x00
+// /////////////////////////////////////////////////////////////////
+// Need Soft Serial defines in config.h
+// /////////////////////////////////////////////////////////////////
+// ex.
+//  /* Configuration of lower interface with the lower layer(hardware) of serial.c */
+//  #define SOFT_SERIAL_PIN ??   // ?? = D0,D1,D2,D3,E6
```

Comment:
Could define the default here:
```suggestion
#ifndef SOFT_SERIAL_PIN 
  #define SOFT_SERIAL_PIN D0   // D0,D1,D2,D3,E6, etc
#endif
```

This way, there is no reason to define it elsewhere.

Reply:
see 
https://github.com/qmk/qmk_firmware/pull/4419#issuecomment-438721337